### PR TITLE
Portland 2022 Draft schedule

### PR DESCRIPTION
**Pull request by plaindocs**
- [x] Double check times with speakers
- [ ] confirm local link works in mail out
- [x] confirm shirt sales status

### CODE REVIEW

**Review by mxsasha — APPROVED** [review]
Looks like a good mixup to me. Didn't look at timing of things otherwise as you all discussed that extensively in the other PR.

File: docs/conf/portland/2022/news/announcing-schedule.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.. post:: Apr 11, 2022
+   :tags: {{year}}, portland-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're just over a month away from the online Portland conference, and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familiar with.
```
Comment (ericholscher) [inline docs/conf/portland/2022/news/announcing-schedule.rst]:
```suggestion
We're just over a month away from the online Portland conference, and we're working hard to make sure it'll be as welcoming and fun as previous years.
```

File: docs/conf/portland/2022/news/announcing-schedule.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Conference platform
+--------------------
+
+Following the success of last year's virtual format, the conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. We're updating our Welcome Wagon guide, to be released soon, so it'll all feel familiar before you've even arrived.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the :doc:`/conf/portland/{{year}}/schedule/` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to continue the new tradition of live Q&A after most talks, this year two speakers will be chatting with the MC after every couple of talks, a little like a mini panel, which will come from your questions but be moderated by our MC.
```
Comment (plaindocs) [inline docs/conf/portland/2022/news/announcing-schedule.rst]: @ericholscher will this convert ok in the mail out? 

```
:doc:`/conf/portland/{{year}}`
```

**Comment by thatdocslady** [comment]
Looks great, a few last-minute questions:

1- Do we want to double-check with WW that they'll do the tours again?
2- Do we want to be extra explicit in the Q&A slots and say something like "Combined Q&A for the previous two talks" or similar?
3- Do we want to align the job fair with the first talk, and do we want to add an entry for ending time?
4- In the past we had the closing address prerecorded and broadcast before the final Q&A, should we do this again?

File: docs/conf/portland/2022/news/announcing-schedule.rst (hunk 2 above)
Comment (ericholscher) [reply docs/conf/portland/2022/news/announcing-schedule.rst]: Yea, we make all links fully qualified (99% sure :)

Comment (ericholscher) [inline docs/conf/portland/2022/news/announcing-schedule.rst]:
```suggestion
The conference website is now updated with the :doc:`/conf/portland/{{year}}/schedule/` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to continue and adapt live Q&A after most talks. This year two speakers will be chatting with the MC each block of 2 talks, a little like a mini panel, which will come from your questions but be moderated by our MC.
```

**Review by ericholscher — COMMENTED** [review]
> 3- Do we want to align the job fair with the first talk, and do we want to add an entry for ending time?

I think yes on both.

**Comment by plaindocs** [comment]
I've addressed most of those. If someone could decide on the the closing address / Q&A thing, and push this out that would be lovely.

**Comment by thatdocslady** [comment]
I vote to have the closing address prerecorded and attached to the final talk.

**Review by thatdocslady — APPROVED** [review]

**Comment by ericholscher** [comment]
🎉

**Pull request merged**